docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,502 @@
-``crate-node repurpose`` command to find and remove excess shard data and table
+First update the settings in the `crate.yml config file` of the node to:
+
+* ``node.master: false``
+* ``node.data: false``
+
+.. NOTE::
+
+  Before you continue with the next step, you need to stop CrateDB.
+
+Run ``crate-node repurpose`` to find and remove excess shard data and table
 metadata:
 
 .. code-block:: console
 
     node$./bin/crate-node repurpose
 
-      WARNING: CrateDB MUST be stopped before running this tool.
-
-    Found 2 tables (2 shards and 2 table meta data) to clean up
-    Use -v to see list of paths and tables affected
-    Node is being re-purposed as no-master and no-data. Clean-up of table data will be performed.
+    Found 2 tables (2 shards and 2 table metadata) to clean up.
+    Use -v to see list of paths and tables affected.
+    Node is being repurposed as no-master and no-data. Clean-up of table data
+    is performed.
 
     Do you want to proceed?
 
-    Confirm [y/N] y
+    Confirm [y/n] y
     Node successfully repurposed to no-master and no-data.
 
+.. _conf-env-re-purposing-nodes-unsafe-bootstrapping:
+
 Unsafe cluster bootstrapping
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Suppose your cluster had five master-eligible nodes and you have permanently
-lost three of them, leaving two nodes remaining.
+You should only run this tool if you have permanently lost half or more
+of the master-eligible nodes in this cluster. Also only run this tool if you
+cannot restore the cluster from a snapshot. If you have multiple surviving
+master-eligible nodes, you should run this tool on the node with the highest
+cluster state (term, version) pair.
+
+Suppose your cluster has had five master-eligible nodes and you have permanently
+lost three of them, leaving two nodes remaining. In such a case you can unsafely
+bootstrap the cluster.
+
+.. NOTE::
 
-* Run the tool on the first remaining node, but answer n at the confirmation step.
+    Before you run ``crate-node unsafe-bootstrap`` you need to stop CrateDB.
+
+.. WARNING::
+
+    This tool can cause arbitrary data loss, and its use should be your last
+    resort.
+
+* Run ``crate-node unsafe-bootstrap`` on the first remaining node
+  and answer ``n`` at the confirmation step.
 
 .. code-block:: console
 
     node_1$ ./bin/crate-node unsafe-bootstrap
 
-      WARNING: CrateDB MUST be stopped before running this tool.
-
     Current node cluster state (term, version) pair is (4, 12)
 
-    You should only run this tool if you have permanently lost half or more
-    of the master-eligible nodes in this cluster, and you cannot restore the
-    cluster from a snapshot. This tool can cause arbitrary data loss and its
-    use should be your last resort. If you have multiple surviving master
-    eligible nodes, you should run this tool on the node with the highest
-    cluster state (term, version) pair.
-
     Do you want to proceed?
 
-    Confirm [y/N] n
+    Confirm [y/n] n
```

Comment:
N is uppercase on purpose, since it is the default option when you just press enter. This is common with command line tools. So it should be ``Confirm [y/N]``